pom.xml

```diff
@@ -1012,9 +1015,10 @@
                         <exclude>${test.files.excludes}</exclude>
                     </excludes>
                     <excludedGroups>ManualTest</excludedGroups>
+                    <runOrder>random</runOrder>
+                    <parallel>true</parallel>
```

Comment:
Same goes for this change

Reply:
The test was effectively `parallel` already, I just made it explicit